src/algokit/cli/project/bootstrap.py

```diff
@@ -102,5 +102,11 @@ def poetry() -> None:
 @bootstrap_group.command(
     "npm", short_help="Runs `npm install` in the current working directory to install Node.js dependencies."
 )
-def npm() -> None:
-    bootstrap_npm(Path.cwd())
+@click.option(
```

Comment:
Does that change apply to `all` as well?

Reply:
`all` already has --ci, --interactive/--no-interactive options where (--ci is equivalent to --no-interactive).